install/example/example.go

```diff
@@ -105,12 +106,12 @@ func fetchFile(url string) (string, error) {
 		return "", fmt.Errorf("received non OK status code while getting a file from url: %s", url)
 	}
 
-	data, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
+	var data bytes.Buffer
+	if _, err := io.Copy(&data, resp.Body); err != nil {
```

Comment:
Is it really necessary in the example? It is not accessed concurrently and we load it whole into the memory anyways so I do not really see any benefits. As `ioutil.ReadAll(resp.Body)` is simpler I think it would be more clear in the example.

Reply:
Even if it is not accessed concurrently, it might happen that the resp.Body is huge.
So, I think it is better to be safe using io.Copy(), as it will only uses fixed 32KB buffer to copy from source to destination until EOF